magento/magento2
# Fix issue #10565

Fix: Creation of a new product with custom attribute set now works again

### Description
https://github.com/magento/magento2/commit/85db4c87491d43fa4d4de59c6fe3d6cc1486835b#diff-620105b6e833499c14f4eb74ce8a4532 introduced some changes wich broke the add new product screen when choosing a custom attribute set with a multiselect attribute in it. This pull request fixes that

### Fixed Issues 
1. magento/magento2#10565: Magento ver. 2.1.8 New Product with Custom attribute set not workink

### Manual testing scenarios
1. Create a new product from admin, select a non-default attribute set
2. Check that the product form loads without JS errors

### Contribution checklist
